greet.js

```diff
@@ -0,0 +1,10 @@
+export function greet(name){
+    const names = ['Thomas','Stefan','Martin'];
+    if (names.includes(name)){
```

Comment:
Awesome Zach. A bit advanced right now. But a very good and easy way to add names to check for. 👍 ❤️